optuna_dashboard/_app.py

```diff
@@ -220,6 +221,10 @@ def get_study_detail(study_id: int) -> dict[str, Any]:
         ) = get_cached_extra_study_property(study_id, trials)
 
         plotly_graph_objects = get_plotly_graph_objects(system_attrs)
+        trials_id2number = {trial._trial_id: trial.number for trial in trials}
+        skipped_trials = [
+            trials_id2number[trial_id] for trial_id in get_skipped_trial_ids(system_attrs)
+        ]
```

Comment:
I think you can write this logic like below:

```suggestion
        skipped_trial_ids = get_skipped_trial_ids(system_attrs)
        skipped_trials = [t.number for t in trials if t._trial_id in skipped_trial_ids]
```